plugins/DynamicsProcessor/DynamicsProcessor.cpp

```diff
@@ -56,7 +56,7 @@ Plugin::Descriptor PLUGIN_EXPORT dynamicsprocessor_plugin_descriptor =
 }
 
 const float DYN_NOISE_FLOOR = 0.00001f; // -100dBFS noise floor
-const double DNF_LOG = 5.0;
+const double DNF_LOG = -1.0;
```

Comment:
Just double-checking you meant this to be -1 not +1

Reply:
Yes, the negative sign was removed from the coefficient calculation function.